Backend/Interview.Test/Units/Processors/RoomActiveStatusPreProcessorTest.cs

```diff
@@ -0,0 +1,59 @@
+using FluentAssertions;
+using Interview.Domain.Events.DatabaseProcessors;
+using Interview.Domain.Repository;
+using Interview.Domain.Rooms;
+using Interview.Domain.Rooms.RoomTimers;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Interview.Test.Units.Processors;
+
+public class RoomActiveStatusPreProcessorTest
+{
+    private readonly RoomActiveStatusPreProcessor _roomActiveStatusPreProcessor;
+
+    public RoomActiveStatusPreProcessorTest()
+    {
+        var databaseContext = new TestAppDbContextFactory().Create(new SystemClock());
```

Comment:
It is better for tests to pass a test Clock to check a specific time in tests. Then the test will not be tied to the current time.